Authorization/Authorization/Presentation/Registration/SignUpViewModel.swift

```diff
@@ -30,15 +30,32 @@ public class SignUpViewModel: ObservableObject {
     }
     
     @Published var fields: [FieldConfiguration] = []
-    
+    var requiredFields: [FieldConfiguration] {
+       fields
+            .filter {
+                $0.field.required &&
+                !$0.field.isHonorCode &&
+                $0.field.type != .checkbox
+            }
```

Comment:
Please format it like 
```
fields.filter {
            $0.field.required &&
            !$0.field.isHonorCode &&
            $0.field.type != .checkbox
        }
```
because on other places the same pattern is being used in this file.